site/docs/concepts/web-app.md

```diff
@@ -0,0 +1,272 @@
+
+The order of the tabs mirrors the order of a basic Data Flow:
+
+import Mermaid from '@theme/Mermaid';
+
+<Mermaid chart={`
+	graph LR;
+		Capture-->Collection;
+        Collection-->Materialization;
+`}/>
+
+While you may choose to [use the tabs in this sequence](../guides/create-dataflow.md), it's not necessary.
+All Flow entities exist individually, outside of the context of complete Data Flow.
+You can use the different pages in the web app to monitor and manage your items in a number of other ways, as described below.
+
+## Captures page
+
+The **Captures** pages shows you a table of existing Flow [captures](./captures.md) to which you have [access](../reference/authentication.md).
+The **New Capture** button is also visible.
+You use the table to monitor your captures.
+
+![](<./webapp-images/capture-page.png>)
+
+**1:** Select all or deselect all.
+
+**2:** Enable, Disable, and Delete buttons. These actions will be applied to the selected table rows. Choose **Disable** to temporarily pause the flow of data, **Enable** to resume, and **Delete** to permanently remove the capture(s).
+
+**3:** Materialize button. When you click this button, you're directed to the **Create materializations** page.
+All the collections of the selected capture(s) will be added to the materialization.
+
+**4:** Filter captures by name.
+
+**5:** Shard status indicator. Shows the status of the task [shard](./advanced/shards.md) that backs this capture.
+
+* **Primary (Green)**: Data is actively flowing through the capture.
+* **Pending (Yellow)**: The capture is attempting to re-connect. Often, you'll see this after you re-enable the capture as Flow backfills historical data.
+* **Failed (Red)**: The capture has failed with an unrecoverable error.
+* **Disabled (White)**: The capture is disabled.
+* **Unknown (Black)**: The web app is unable to determine shard status. Usually, this is due to a temporary connection error.
+
+**6:** Capture name. The full name is shown, including all [prefixes](./catalogs.md#namespace).
+
+**7:** Capture type. The icon shows the type of source system data is captured from.
+
+**8:** Associated collections. The **Writes to** column shows all the collections to which the capture writes data. For captures with a large number of collections, hover over this column and scroll to view the full list.
+
+**9:** Publish time. Hover over this value to see the exact time the capture was first published.
+
+**10:** Options. Choose to **View Details** or **Edit Specification**.
+
+### Detail view
+
+When you click **View Details** for a capture, the **Status** and **Specification** viewers are revealed.
+
+The **Status** window shows the full identifier of the shard that backs your capture. If there's an error, you can view its logs.
+
+In the **Specification** window, you can view the specification of the capture itself, as well as each collection to which it writes.
```

Comment:
Should we highlight the fact that the left-hand pane of the _Specification window_ can be resized in the event a long catalog name is truncated? This was a point of tension in the UI in the past; people were not aware that you can horizontally resized each side of the _Specification window_. Some tweaks were made to the divider to make this more obvious, but it may be worth noting.

Reply:
Added a Tip here, but I didn't make a note in every location where a two-paneled view is used... This is because I do think those UX tweaks made it pretty obvious. If people are still confused going forward I can call this out more frequently.

Reply:
Okay, excellent. Glad to hear the changes made to that divider improved this UX.